tools/autograd/derivatives.yaml

```diff
@@ -2953,3 +2953,6 @@
 - name: _reshape_copy(Tensor self, SymInt[] size) -> Tensor
   self: grad.reshape_symint(self.sym_sizes())
   result: auto_linear
+
+- name: all_reduce(Tensor self, int group_id, str reduce_op) -> Tensor
+  self: at::ones_like(self)
```

Comment:
That sounds wrong!
This should be a `all_scatter()` no?
If you don't want it to be differentiable right now, mark it as such with `non_differentiable` (see doc at the top of this file for details).